_layouts/default.html

```diff
@@ -13,6 +13,9 @@
       <link rel="stylesheet" href="{{ site.baseurl }}/css/main.css">
       <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
       <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Zilla+Slab|Zilla+Slab+Highlight:400,700">
+
```

Comment:
oh sorry, I told you the wrong thing. As this is only used in home.html, it should also live there. Sorry about that. :(

You can add it at the top of https://github.com/mozilla/fx-webcompat-campaign/blob/master/_layouts/home.html#L4.